<!--edit by lichunjiing 2020_12_23 -->
<launch>
  <param name="/use_sim_time" value="true" />
  <arg name="rviz" default="true" />

  <arg name="cfg_file_path" default="/home/alorli/WORK_SPACE_LCJ/catkin_ws/src/lidar_mapping/cfg" />

  <node name="lidar_mapping_node"  pkg="lidar_mapping"  type="lidar_mapping_node" output="screen" respawn="true"> 
    <param name="cfg_file_path" value="$(arg cfg_file_path)" />
  </node>

  <group if="$(arg rviz)">
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lidar_mapping)/rviz/lidar_mapping.rviz" />
  </group>

</launch>
